### PR TITLE
Ref update

### PR DESCRIPTION
**Pull request by seanturner**
Now refer to I-D.lamps-cms-ml-dsa.

### CODE REVIEW

File: draft-ietf-lamps-dilithium-certificates.md [hunk 1]
```diff
@@ -575,10 +575,10 @@ single OCTET STRING.
 This appendix includes the ASN.1 module {{X680}} for the ML-DSA.  Note that
 as per {{RFC5280}}, certificates use the Distinguished Encoding Rules; see
 {{X690}}. This module imports objects from {{RFC5912}} and
-{{!I-D.salter-lamps-cms-ml-dsa}}.
+{{!I-D.lamps-cms-ml-dsa}}.
```
Comment (danvangeest) [inline draft-ietf-lamps-dilithium-certificates.md:578]:
```suggestion
{{!I-D.ietf-lamps-cms-ml-dsa}}.
```

File: draft-ietf-lamps-dilithium-certificates.md [hunk 2]
```diff
@@ -575,10 +575,10 @@ single OCTET STRING.
...
 
 <aside markdown="block">
-RFC EDITOR: Please replace TBD2 with the value assigned by IANA during the publication of {{I-D.salter-lamps-cms-ml-dsa}}. Also please replace {{I-D.salter-lamps-cms-ml-dsa}} in the module with a reference to the published RFC.
+RFC EDITOR: Please replace TBD2 with the value assigned by IANA during the publication of {{I-D.lamps-cms-ml-dsa}}. Also please replace {{I-D.lamps-cms-ml-dsa}} in the module with a reference to the published RFC.
```
Comment (danvangeest) [inline draft-ietf-lamps-dilithium-certificates.md:581]:
```suggestion
RFC EDITOR: Please replace TBD2 with the value assigned by IANA during the publication of {{I-D.ietf-lamps-cms-ml-dsa}}. Also please replace {{I-D.ietf-lamps-cms-ml-dsa}} in the module with a reference to the published RFC.
```

**Review by danvangeest — CHANGES_REQUESTED** [review]
Please update the reference in the ASN.1 module as well.

**Review by bwesterb — APPROVED** [review]

**Comment by seanturner** [comment]
Good catch about the module!

File: X509-ML-DSA-2024.asn [hunk 3]
```diff
@@ -19,7 +19,7 @@ PUBLIC-KEY, SIGNATURE-ALGORITHM
 
 sa-ml-dsa-44, sa-ml-dsa-65, sa-ml-dsa-87,
 pk-ml-dsa-44, pk-ml-dsa-65, pk-ml-dsa-87
-  FROM ML-DSA-Module-2024 -- From [I-D.salter-lamps-cms-ml-dsa]
+  FROM ML-DSA-Module-2024 -- From [I-D.lamps-cms-ml-dsa]
```
Comment (seanturner) [inline X509-ML-DSA-2024.asn:22]:
```suggestion
  FROM ML-DSA-Module-2024 -- From [I-D.ietf-lamps-cms-ml-dsa]
```

**Pull request merged**